src/main/java/uk/gov/hmcts/reform/idam/web/strategic/PolicyService.java

```diff
@@ -104,4 +109,38 @@ private boolean checkNoActionsBlockingUser(ResponseEntity<EvaluatePoliciesRespon
         return true;
     }
 
+    /**
+     * Sanitise and return request ips in a list of strings
+     * Examples:
+     * "1.1.1.1" => ["1.1.1.1"]
+     * "51.140.12.192:59286" => ["51.140.12.192"]
+     * "51.140.12.192:59286, 10.97.64.4:59250, 10.97.66.7:57249" => ["51.140.12.192", "10.97.64.4", "10.97.66.7"]
```

Comment:
In that case - does AM check that _none_ of the IPs in the list is in the range of allowed IPs?

Reply:
we can have multiple rules
using AND or OR and NOT we can achieve what we need 👍